posthog/queries/funnels/test/test_breakdowns_by_current_url.py

```diff
@@ -0,0 +1,149 @@
+class TestBreakdownsByCurrentURL(ClickhouseTestMixin, APIBaseTest):
+    def setUp(self):
+        super().setUp()
+        journey = {
+            "person1": [
+                # no trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com", "$pathname": ""},
+                },
+                # trailing question mark
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com?", "$pathname": "?"},
+                },
+                {
+                    "event": "terminate funnel",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                },
+            ],
+            "person2": [
+                # trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/", "$pathname": "/"},
+                },
+                # trailing hash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com#", "$pathname": "#"},
+                },
+                {
+                    "event": "terminate funnel",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                },
+            ],
+            "person3": [
+                # no trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home", "$pathname": "/home"},
+                },
+                {
+                    "event": "terminate funnel",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                },
+            ],
+            "person4": [
+                # trailing slash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home/", "$pathname": "/home/"},
+                },
+                # trailing hash
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home#", "$pathname": "/home#"},
+                },
+                # all the things
+                {
+                    "event": "watched movie",
+                    "timestamp": datetime(2020, 1, 2, 12, 1),
+                    "properties": {"$current_url": "https://example.com/home/?#", "$pathname": "/home/?#"},
```

Comment:
Is the idea to only handle 'empty' stuff, i.e. `/home/?q=1` and `/home/?q=2` are different, but `/home/`, `/home/?` are the same?

Reply:
Yep